airflow/cli/commands/webserver_command.py

```diff
@@ -396,6 +395,9 @@ def webserver(args):
             'python:airflow.www.gunicorn_config',
         ]
 
+        if args.opentelemetry:
+            run_args = ['opentelemetry-instrument'] + run_args
+
         if args.access_logfile:
             run_args += ['--access-logfile', str(args.access_logfile)]
 
```

Comment:
Also here it seems there is a problem. When I run ./breeze start-airflow --integration jaeger, in the airflow trigger it says there's a Name error and highlights this line args.opentelemetry

Reply:
You need to add ARGS_OPENTELEMETRY to the webseserver command (the same file but way down).

```
    ActionCommand(
        name='webserver',
```
....

Reply:
ok

Reply:
This is only needed for webserver (for now) to implement the gunicorn workaround.

I tink for now You can add it as `OPENTELEMETRY-READM.md` in the main directory of Airflow. We can move it later.